pkg/provisioning/validate_test.go

```diff
@@ -0,0 +1,180 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package provisioning
+
+import (
+	"testing"
+
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/conduitio/conduit/pkg/foundation/multierror"
+	"github.com/matryer/is"
+)
+
+func TestValidator_ConnectorMandatoryField1(t *testing.T) {
```

Comment:
Nitpick and maybe even a preference thing: it would be good that whatever is printed in the logs, when a test fails, is more or less enough to know what test case failed. We can achieve that by using an appropriate test method name or IIUC by setting a test name. Now, `TestValidator_ConnectorMandatoryField1` is already pretty clear, and maybe we can leave it like that. What I'm suggest is having something like `TestValidator_ConnectorMandatoryField_TypeName`.

Reply:
This would be a great opportunity to create [table driven test](https://github.com/golang/go/wiki/TableDrivenTests) 🙂 Something like `TestValidator_MandatoryFields` and a test case for each mandatory field.